### PR TITLE
PYIC-7164 Update page to include context (repeatFraudCheck) check for fast follow.

### PR DESCRIPTION
**Pull request by rpayal**
## Proposed changes
Update page to include context (repeatFraudCheck) check for fast follow.

### Issue tracking

- [PYIC-7164](https://govukverify.atlassian.net/browse/PYIC-7164)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[PYIC-7164]: https://govukverify.atlassian.net/browse/PYIC-7164?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -1287,10 +1287,13 @@
       "header": "You were not able to update your details with the GOV.UK ID check app",
       "content": {
         "paragraph1": "You can still use the service if your details are out of date.",
+        "paragraph1RepeatFraudCheck": "You will not be able to use your GOV.UK One Login to access the service.",
         "subHeading": "What would you like to do?",
         "formRadioButtons": {
           "continueOption": "Continue to the service without updating your details",
           "continueOptionHint": "Your proof of identity is valid even if your details have changed.",
+          "backOption": "Go back to the service you are trying to use",
```
Comment (thebauSoftwire) [inline src/locales/en/translation.json]: I think this should be "Go back to the service you _were_ trying to use" from the [designs](https://www.figma.com/design/nglBHtbzJYEeST43Iu1jW3/Identity-Pod---UCD-Hive?node-id=26170-21100&node-type=FRAME&t=REOnSSVibUkr7yf8-0)

**Review by thebauSoftwire — DISMISSED** [review]
Nice! Just one tiny comment :)

File: src/views/ipv/page/update-details-failed.njk [hunk 2]
```diff
@@ -11,20 +11,32 @@
 
 {% block content %}
   <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}">{{ 'pages.updateDetailsFailed.header' | translate }}</h1>
-  <p class="govuk-body">{{ 'pages.updateDetailsFailed.content.paragraph1' | translate }}</p>
+  {% if context == "repeatFraudCheck" %}
+    <p class="govuk-body">{{ 'pages.updateDetailsFailed.content.paragraph1RepeatFraudCheck' | translate }}</p>
```
Comment (sam803) [inline src/views/ipv/page/update-details-failed.njk]: you should be able to use `translateWithContext(context)` here and remove the if/else

File: src/locales/en/translation.json (hunk 1 above)
Comment (rpayal) [reply src/locales/en/translation.json]: done

File: src/views/ipv/page/update-details-failed.njk (hunk 2 above)
Comment (rpayal) [reply src/views/ipv/page/update-details-failed.njk]: Done

**Review by sam803 — APPROVED** [review]

**Pull request merged**